### DISCUSSION TITLE
Parallel commands with useNx

### DISCUSSION THREAD

**Discussion by lukasbach** [lerna/lerna#3192] [Q&A] [answered]
I have a lerna monorepo with a command that should run in all packages simultanuously, i.e. run a typescript watch server in all packages and keep them running indefinitely until they are stopped forcefully, with the code `lerna run start --scope=web --include-dependencies --parallel`. That works fine, but not anymore with switching the `useNx` flag to true, since then it only seems to run the first 3 packages (since the parallel max count is set to three) and then stalls because these 3 packages never end (by design). What is the intended way of getting these always-running commands to work that should run on more than 3 packages with the useNx flag?

Also, slightly unrelated, if I would switch the command from lerna to nx run-many, is there a way to replace the `--include-dependencies` with something supported by nx so I can only specify one package and it will automatically run all its transitive dependencies?

Thanks in advance!

**Comment by JamesHenry** [comment, marked as answer]
Hi @lukasbach as you hopefully saw, this was a bug and was addressed in https://github.com/lerna/lerna/pull/3205 and is available in the latest release! Many thanks
